### DISCUSSION TITLE
Add Social Icons

### DISCUSSION THREAD

**Discussion by Pro-official** [tailwindlabs/heroicons#716] [Ideas]
Should add social icons in Heroicons. Every developer needs to work with social icons now and then. I think it's essential to add these social icons here.

**Comment by hamzaelmaghari** [comment]
Could be better if added social networks icons.

**Comment by zefir-git** [comment]
I don't think this is a good idea at all. Adding brands to an iconpack means you have to watch for updates, ensure brand usage compliance, and it generally convolutes the iconpack as more brands are added. Not to mention that brand icons do not fit the style of the rest of the iconpack.

Instead, use Simpleicons.org for SVG brand icons. That project is specifically for brand icons only.
